cmake/default_build.cmake

```diff
@@ -0,0 +1,48 @@
+add_subdirectory(${LIBDAISY_DIR} libdaisy)
+add_subdirectory(${DAISY_SP_DIR} DaisySP)
```

Comment:
Can we make DaisySP optional?

Reply:
this default build is supposed to work as a copy of the `Makefile` build;
you can also use it as a reference to make your own :)

Reply:
The Makefile accepts if you don't have DAISY_SP_DIR set - it will not link against daisySP then

Reply:
hm, didn't notice that; will add the check then :)